.github/workflows/main.yml

```diff
@@ -161,3 +161,31 @@ jobs:
         with:
           name: cypress-videos
           path: cypress/videos
+
+  link-check:
+    name: 'Check links'
+
+    needs: build_deploy_test
+
+    if: github.event_name != 'push' && github.ref_name != 'main'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+
```

Comment:
I dont think we need nodejs here.  Actions run without explicit installation of their environment.